view/adminhtml/web/template/view/certificates-grid.html

```diff
@@ -0,0 +1,54 @@
+<!-- ko if: shouldShowWarning -->
+<p class="message">
```

Comment:
Add translation. This paragraph should all be on a single line to allow for CSV translation.